backend/src/v1/routes/announcement-routes.ts

```diff
@@ -24,11 +24,40 @@ import {
   PatchAnnouncementsType,
 } from '../types/announcements';
 import { omit } from 'lodash';
+import { DateTimeFormatter, ZonedDateTime } from '@js-joda/core';
 
 const router = Router();
 
+type SsoRequest = ExtendedRequest & { query: AnnouncementQueryType };
+
 router.get(
   '',
+  authenticateAdmin(false),
+  async (req: SsoRequest, res, next) => {
+    if (!req?.user?.admin_user_id) {
```

Comment:
can we clear the query parameters and then add this in.

Reply:
cleared all query parameters